scrapers/Waybig.yml

```diff
@@ -0,0 +1,67 @@
+name: Waybig
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - waybig.com/video/
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - waybig.com/pornstars/
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@name="twitter:title"]/@content
+      Date:
+        selector: //i[@class="far fa-calendar"]/following-sibling::text()
+        postProcess:
+          - replace:
+              - regex: ^(\d{4}-\d{2}-\d{2}).+
+                with: $1
+          - parseDate: 2006-01-02
+      Details:
+        selector: //div[@class="expand-inner"]
+      Tags:
+        Name: //div[contains(text(),"Tags:")]/following-sibling::a
+      Performers:
+        Name:
+          selector: //div[@class="models-box"]/a
```

Comment:
```yaml
        Name: //div[@class="models-box"]/a
        URL: //div[@class="models-box"]/a/@href
```
since you already made a performer scraper adding the performer url here is very useful